test/200-bridge-firewalld.bats

```diff
@@ -329,3 +329,71 @@ function setup() {
     expected_rc=1 run_netavark -f ${TESTSDIR}/testfiles/invalid-port.json setup $(get_container_netns_path)
     assert_json ".error" "invalid host ip \"abcd\" provided for port 8080" "host ip error"
 }
+
+@test "$fw_driver - test firewalld reload" {
+    setup_firewalld
```

Comment:
this must be dropped now that it is part of the firewalld file

Reply:
Thanks! The change is in the latest PR now.